src/main/resources/javascript/locales/de.json

```diff
@@ -1,13 +1,5 @@
 {
   "label": {
-    "contentManager": {
-      "leftMenu" : {
-        "manage" : {
-          "contentReports" : {
-            "title" : "Content Reports"
-          }
-        }
-      }
-    }
+    "contentreports": "Content Reports"
```

Comment:
Do you have a task or something to track the missing RB ?

Reply:
I don't know, but I just moved the labels from the old jsp to here .. so did not add or remove new ones